pkg/cmd/clientConfig_test.go

```diff
@@ -0,0 +1,91 @@
+package cmd
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	kFake "k8s.io/client-go/kubernetes/fake"
+)
+
+type args struct {
```

Comment:
Not sure on this, why have a type like this in the test?

Reply:
https://github.com/aerogear/mobile-cli/blob/master/pkg/cmd/clientConfig.go#L33, but agree that I have only one instance of of same client. Need change to function